tests/test_processing.py

```diff
@@ -1,12 +1,23 @@
+import numpy as np
+import pytest
+
 from glidertools.processing import (
     calc_backscatter,
     calc_fluorescence,
     calc_oxygen,
     calc_par,
     calc_physics,
 )
+from tests.test_physics import dat
 
 
-def test_dummy():
-    """WE REALLY NEED TO ADD TESTS!!! THESE JUST TEST THE BASIC IMPORT!!!"""
-    assert 1 == 1
+def test_calc_oxygen():
+    o2ml, o2pc, o2au = calc_oxygen(
```

Comment:
shouldn't this be `o2aou` instead of `a2au`.

Reply:
sorry maybe I also don't get it! 🙄

Reply:
Yeah it should be, typo on my part. More importantly, we need to know if the values make any sense, but I'm not confident in testing that